src/main/scala/com/decodified/scalassh/Command.scala

```diff
@@ -18,11 +18,26 @@ package com.decodified.scalassh
 
 import java.io.{ByteArrayInputStream, File, FileInputStream, InputStream}
 import net.schmizz.sshj.connection.channel.direct.Session
-
-final case class Command(command: String, input: CommandInput = CommandInput.NoInput, timeout: Option[Int] = None)
+trait Command {
+  val command: String
+  val input: CommandInput  = CommandInput.NoInput
+  val timeout: Option[Int] = None
+  val safe: Boolean
```

Comment:
I'd say that "safe" in the context of a `Command` would usually be understood as sth like "safe to run", as in "doesn't do any harm".
What we are looking for here is sth more like "safe to be logged in the clear".

Reply:
That was quite ambiguous, the new value that can be passed into `Command` is `safeToLog` and is still opt in.